tests/legacy-cli/e2e/tests/generate/lint-fix.ts

```diff
@@ -14,13 +14,10 @@ export default function () {
     }
   }`;
 
-  // TODO(architect): reenable after figuring out what happens to --lint-fix.
-  return;
-
   return Promise.resolve()
 
     // setup a double-quote tslint config
-    .then(() => writeFile('src/app/tslint.json', nestedConfigContent))
+    .then(() => writeFile('tslint.json', nestedConfigContent))
```

Comment:
Nit: being at toplevel means it's not a tested config anymore.

Reply:
Done.